File: Repo_Redes/obligatorio/InstaPhotoAdministrativeWebPage/src/app/models/Log.ts
export class LogModel {
Author: String;
DateTime: Date;
Type: Number;
TypeSeverity: Number;

  constructor(Author: String,DateTime: Date,Type: Number,TypeSeverity: Number){
    this.Author = Author;
    this.DateTime = DateTime;
    this.Type = Type;
    this.TypeSeverity = TypeSeverity;
  }
}

File: Repo_Redes/obligatorio/SharedLog/Log.cs
using System;
using SharedLogConstant;

namespace SharedLog
{
    public class Log
    {
        public string Author { get; set; }
        public DateTime Date { get; set; }
        public LogEnum Type { get; set; }
        public LogEnumSeverity TypeSeverity { get; set; }
        public Log()
        {
            Date = DateTime.Now;
        }
        public Log(string oneAuthor, LogEnum oneType,LogEnumSeverity oneTypeSeverity)
        {
            Author = oneAuthor;
            Date = DateTime.Now;
            Type = oneType;
            TypeSeverity = oneTypeSeverity;
        }

        public string ToString()
        {
            return Author + " realizo:" + Type + " - " + Date + " Con Severidad:" + TypeSeverity;
        }
    }
}